ballerina/file_csv_io.bal

```diff
@@ -21,30 +33,44 @@
 # + path - The CSV file path
 # + skipHeaders - Number of headers, which should be skipped prior to reading records
 # + return - The entire CSV content in the channel as an array of string arrays or an `io:Error`
-public isolated function fileReadCsv(string path, int skipHeaders = 0) returns string[][]|Error {
-    return channelReadCsv(check openReadableCsvFile(path, COMMA, DEFAULT_ENCODING, skipHeaders));
-}
+# + returnType - The type of the return value (string[] or map<anydata>)
+public isolated function fileReadCsv(string path, int skipHeaders = 0, typedesc<string[]|map<anydata>> returnType = <>) returns returnType[]|Error = @java:Method{
+    name: "fileReadCsv",
+    'class: "io.ballerina.stdlib.io.nativeimpl.CsvChannelUtils"
+} external;
 
 # Read file content as a CSV.
 # ```ballerina
-# stream<string[], io:Error?>|io:Error content = io:fileReadCsvAsStream("./resources/myfile.csv");
+# stream<returntype, io:Error?>|io:Error content = io:fileReadCsvAsStream("./resources/myfile.csv");
 # ```
 # + path - The CSV file path
 # + return - The entire CSV content in the channel a stream of string arrays or an `io:Error`
-public isolated function fileReadCsvAsStream(string path) returns stream<string[], Error?>|Error {
-    return channelReadCsvAsStream(check openReadableCsvFile(path));
-}
+# + returnType - The type of the return value (string[] or map<anydata>)
+public isolated function fileReadCsvAsStream(string path, typedesc<string[]|map<anydata>> returnType = <>) returns stream<returnType, Error?>|Error = @java:Method{
+    name: "createCsvAsStream",
+    'class: "io.ballerina.stdlib.io.nativeimpl.CsvChannelUtils"
+} external;
+
+// # Read file content as a CSV.
+// # ```ballerina
+// # stream<string[], io:Error?>|io:Error content = io:fileReadCsvAsStream("./resources/myfile.csv");
+// # ```
+// # + path - The CSV file path
+// # + return - The entire CSV content in the channel a stream of string arrays or an `io:Error`
+// public isolated function fileReadCsvAsStream(string path) returns stream<string[], Error?>|Error {
+//     return channelReadCsvAsStream(check openReadableCsvFile(path));
+// }
```

Comment:
Same as above. Don't commented out the code

Reply:
done